samples/snippets/csharp/VS_Snippets_CLR/AssemblyName_CodeBase/CS/assemblyname_codebase.cs

```diff
@@ -67,8 +67,8 @@ public static void Main()
       myAssemblyName.CodeBase = Directory.GetCurrentDirectory();
       // Set the culture information of the assembly to 'English-American'.
       myAssemblyName.CultureInfo = new CultureInfo("en-US");
-      // Set the hash algoritm to 'SHA1'.
-      myAssemblyName.HashAlgorithm = AssemblyHashAlgorithm.SHA1;
+      // Set the hash algorithm to 'SHA256'.
+      myAssemblyName.HashAlgorithm = AssemblyHashAlgorithm.SHA256;

```

Comment:
Another triplet of AssemblyName.HashAlgorithm being updated.  It might work, I just know that some parts of this system are effectively limited to MD5 and SHA1... and I don't know if this is one of them or not.  (We could, of course, update them, and if we get feedback it doesn't work, then change them back)